build/publish-mrt.yml

```diff
@@ -105,7 +105,6 @@ steps:
   inputs:
     SourceFolder: '${{ parameters.MRTSourcesDirectory }}\packaging'
     Contents: |
-      MrtCore.props      
```

Comment:
also need to remove the native one.

Reply:
the native one does something native specific via MrtCore.C.props, so I think we still need it